sonar-db/src/main/java/org/sonar/db/rule/RuleDao.java

```diff
@@ -132,11 +103,11 @@ private IdToDto(RuleMapper mapper) {
   }
 
   public List<RuleParamDto> selectRuleParamsByRuleKeys(DbSession session, List<RuleKey> ruleKeys) {
-    return executeLargeInputs(ruleKeys, new KeyToRuleParamDto(mapper(session)));
+    return executeLargeInputs(ruleKeys, partitionOfKeys -> mapper(session).selectParamsByRuleKeys(partitionOfKeys));
   }
 
   public List<RuleParamDto> selectRuleParamsByRuleIds(DbSession dbSession, List<Integer> ruleIds) {
-    return DatabaseUtils.executeLargeInputs(ruleIds, new IdToRuleParamDto(mapper(dbSession)));
+    return executeLargeInputs(ruleIds, partitionOfIds -> mapper(dbSession).selectParamsByRuleIds(partitionOfIds));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)